packages/admin/cms-admin/src/common/SearchInput.tsx

```diff
@@ -93,16 +93,16 @@ export const SearchInput = ({
             endAdornment={
                 internalQuery ? (
                     <InputAdornment position="end">
-                        <Typography>
+                        <Typography variant="body2" sx={{ marginRight: "10px" }}>
```

Comment:
IMO we should use the spacing-system whenever possible:

```suggestion
                        <Typography variant="body2" sx={{ marginRight: 2 }}>
```